### PR TITLE
No code indexing

### PR DESCRIPTION
**Pull request by ofermend**
Added ability in docs_crawler to remove code segments from HTML content before sending to Vectara
IF enabled, it removes any text associated with <code> or <script> tags

### CODE REVIEW

File: core/indexer.py [hunk 1]
```diff
@@ -31,14 +31,15 @@ class Indexer(object):
         corpus_id (int): ID of the Vectara corpus to index to.
         api_key (str): API key for the Vectara API.
     """
-    def __init__(self, cfg: OmegaConf, endpoint: str, customer_id: str, corpus_id: int, api_key: str, reindex: bool = True) -> None:
+    def __init__(self, cfg: OmegaConf, endpoint: str, customer_id: str, corpus_id: int, api_key: str, reindex: bool = True, remove_code: bool = True) -> None:
```
Comment (ofermend) [inline core/indexer.py:34]: The "remove_code" flag is generic in indexer, but only used in docs_crawler at this point in time.

File: core/indexer.py [hunk 2]
```diff
@@ -31,14 +31,15 @@ class Indexer(object):
...
         self.cfg = cfg
         self.endpoint = endpoint
         self.customer_id = customer_id
         self.corpus_id = corpus_id
         self.api_key = api_key
         self.reindex = reindex
-        self.timeout = 30
+        self.remove_code = remove_code
+        self.timeout = cfg.vectara.get("timeout", 30)
```
Comment (ofermend) [inline core/indexer.py:42]: making timeout configurable.

**Comment by justinhayes** [comment]
This change looks good to me. Approved.

**Review by cjcenizal — COMMENTED** [review]
It would be great to add unit tests for these utility functions. Unit tests will act as documentation of expected behavior and help us guard against regressions.

**Pull request merged**